example/src/screens/HomeScreen/index.tsx

```diff
@@ -0,0 +1,175 @@
+import { useNavigation } from '@react-navigation/native';
+import type { VFC } from 'react';
+import React, { useCallback, useState } from 'react';
+import type { LayoutChangeEvent } from 'react-native';
+import {
+  Modal,
+  Platform,
+  RefreshControl,
+  StatusBar,
+  StyleSheet,
+  Text,
+  View,
+  useWindowDimensions,
+} from 'react-native';
+import { TabbedHeaderPager } from 'react-native-sticky-parallax-header';
+
+import type { User } from '../../assets/data/cards';
+import { QuizListElement, UserModal } from '../../components';
+import { colors, screenStyles } from '../../constants';
+
+import { EXAMPLES, ExampleLink } from './ExampleLink';
+import { TABS, users } from './data';
+
+const wait = (timeout: number) =>
+  new Promise((resolve) => {
+    setTimeout(resolve, timeout);
+  });
+
+const HomeScreen: VFC = () => {
+  const navigation = useNavigation();
+  const { height: windowHeight } = useWindowDimensions();
+
+  const [refreshing, setRefreshing] = useState<boolean>(false);
+  const [modalVisible, setModalVisible] = useState<boolean>(false);
+  const [userSelected, setUserSelected] = useState<User>();
+  const [contentHeight, setContentHeight] = useState<{ [key: string]: number }>({});
+
+  const openUserModal = (user: User) => {
+    setUserSelected(() => {
+      setModalVisible(true);
+
+      return { ...user };
+    });
+  };
+
+  const onRefresh = useCallback(() => {
+    setRefreshing(true);
+
+    wait(2000).then(() => {
+      setRefreshing(false);
+    });
+  }, []);
+
+  const calcMargin = (title: string): number => {
+    let marginBottom = 50;
+
+    if (contentHeight[title]) {
+      const padding = 24;
+      const isBigContent = windowHeight - contentHeight[title] < 0;
+
+      if (isBigContent) {
+        return marginBottom;
+      }
+
+      const headerHeight = 92;
+
+      marginBottom = windowHeight - padding * 2 - headerHeight - contentHeight[title];
+
+      return marginBottom > 0 ? marginBottom : 0;
+    }
+
+    return marginBottom;
+  };
+
+  const onLayoutContent = (title: string) => (e: LayoutChangeEvent) => {
+    const newHeight = e.nativeEvent.layout.height;
+
+    setContentHeight((prevHeight) => {
+      return { ...prevHeight, [title]: newHeight };
+    });
+  };
+
+  return (
+    <>
+      <StatusBar barStyle="light-content" backgroundColor={colors.primaryGreen} translucent />
+      <TabbedHeaderPager
+        containerStyle={screenStyles.stretchContainer}
+        backgroundColor={colors.primaryGreen}
+        tabsContainerBackgroundColor={colors.secondaryGreen}
+        rememberTabScrollPosition
+        logo={require('../../assets/images/logo.png')}
```

Comment:
Requires like this one could be moved to the `index.ts` file for the images like

```JavaScript
import { logo } from '../../assets/images/index.ts'
```

the `index.ts` file could look like

```JavaScript
export const logo = require('./logo.png')
```

and then you could pass it like

```JavaScript
logo={logo}
```